components/print/cups/Makefile

```diff
@@ -54,37 +58,19 @@ CONFIGURE_OPTIONS +=	--with-smfmanifestdir=/lib/svc/manifest/application
 CONFIGURE_OPTIONS +=	--with-printcap=/etc/printers.conf
 CONFIGURE_OPTIONS +=	--with-cups-user=lp
 CONFIGURE_OPTIONS +=	--with-cups-group=lp
-CONFIGURE_OPTIONS +=	--with-php=no
+CONFIGURE_OPTIONS +=	--disable-systemd
 CONFIGURE_OPTIONS +=	--disable-static
-CONFIGURE_OPTIONS +=	--disable-gnutls
-CONFIGURE_OPTIONS +=	--disable-libusb
-CONFIGURE_OPTIONS +=	--enable-openssl
 CONFIGURE_OPTIONS +=	--enable-dbus
 CONFIGURE_OPTIONS +=	--enable-threads
 CONFIGURE_OPTIONS += 	--with-archflags="$(CC_BITS)"
 
-CONFIGURE_OPTIONS += 	"ac_cv_search_getifaddrs=no"
-CONFIGURE_OPTIONS += 	"ac_cv_func_statfs=no"
-
-$(BUILD_DIR)/%/.installed:	ENV +=	DSTROOT="$(PROTO_DIR)"
-
-PATCH_LEVEL =	0
+#$(BUILD_DIR)/%/.installed:	ENV +=	DSTROOT="$(PROTO_DIR)"
 
 COMPONENT_PRE_CONFIGURE_ACTION = \
  $(CLONEY) $(SOURCE_DIR) $(@D) ; \
  find $(@D) -name '*.in' | sed -e 's/.in$$//' | xargs $(RM) ; \
  ( cd $(SOURCE_DIR) ; autoconf -f )
 
-# common targets
-
-$(INSTALL_32):	$(INSTALL_64)
-
-build:		$(BUILD_32_and_64)
-
-install:	$(INSTALL_32_and_64)
-
-test:		$(NO_TESTS)
-
 # Build dependencies
 REQUIRED_PACKAGES += image/library/libpng16
 REQUIRED_PACKAGES += image/library/libtiff
```

Comment:
You need to re-run gmake REQUIRED_PACKAGES.
openssl is disabled/not supported and shouldn't appear in the list of required packages anymore. Then pkg5 will automatically updated, too.